test/src/main/webapp/META-INF/context.xml

```diff
@@ -101,24 +101,30 @@
  -->
  	<!-- pinGlobalTxToPhysicalConnection does not work the same for mariadb as it does for mysql, therefor still locking problems under XA for mariadb -->
 	
-
-	<Resource 
+	<!-- Use with BTM -->
+ 	<Resource 
 		name="jdbc/ibis4test-postgres-xa" 
-		factory="org.apache.naming.factory.BeanFactory"
+		factory="org.postgresql.xa.PGXADataSourceFactory"
 		type="org.postgresql.xa.PGXADataSource" 
-		URL="jdbc:postgresql://host.docker.internal:5432/testiaf"
+		url="jdbc:postgresql://host.docker.internal:5432/testiaf"
 		user="${testiaf_user/username:-testiaf_user}"
+		username="${testiaf_user/username:-testiaf_user}"
 		password="${testiaf_user/password:-testiaf_user00}"
+		portNumber="5432"
+		serverName="host.docker.internal"
+		databaseName="testiaf"
 	/>
 
+	<!-- Use with default Tomcat pooling -->
 	<Resource name="jdbc/ibis4test-postgres" auth="Container"
-		type="javax.sql.DataSource" 
-		driverClassName="org.postgresql.Driver"
-		url="jdbc:postgresql://host.docker.internal:5432/testiaf"
+		type="javax.sql.XADataSource"
```

Comment:
is het echt nodig een XA datasource te gebruiken? kan je een PGDataSourceFactory gebruiken?